<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Climate Control</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="airFlowDirection">
      <property name="title">
       <string>Air Flow Direction</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="cb_windshield">
         <property name="text">
          <string>Windshield</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_dashboard">
         <property name="text">
          <string>Dashboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_floor">
         <property name="text">
          <string>Floor</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_airCondition">
      <property name="text">
       <string>Air Condition</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_heater">
      <property name="text">
       <string>Heater</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cb_airRecirculation">
      <property name="text">
       <string>Air Recirculation</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
